### PR TITLE
add a guide on contributing

### PR DESCRIPTION
**Pull request by pensjonatus**
resolve #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,186 @@
...
+If you selected to keep your codespace, you can return to your VS Code later.
+It's a good idea to configure it now, so your work goes more smoothly in the
+future.
+
+Install the following extensions:
+
+- Code Spell Checker
+- Polish - Code Spell Checker
+- ESLint
+- GitHub Actions
```
Comment (andrzejniewiaro) [inline README.md:179]: There are four items with "GitHub Actions" name to choose from. I used my powers of deduction and selected the extension with Github icon ;) 
That's nitpicky I know.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,186 @@
...
+
+**Prerequisite:** Before you add your first post, set up your codespace and VS
+Code. See the relevant sections below.
+
+To add a post:
+
+0. Create a branch. You may be working from a GitHub issue. If so,
+   [read about creating branches here](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-a-branch-for-an-issue).
+
+   If you are not working form a GitHub issue (why not? 🤷), you can manually
```
Comment (andrzejniewiaro) [inline README.md:25]: Replace "working form" with "working from".

A stylistic note on the conversational bits like "why not?"- it reads ok the first time, it does not read as well if you have to re-read the section. You could say that there is the preferred way - starting with a Github issue - and the acceptable way - manual branch creation.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,186 @@
...
+The easiest way to contribute to the site, is to set up a GitHub codespace. This
+allows you to run VSCode in the browser, run a site preview, create branches and
+commits, etc.
+
+You can save a codespace and keep it for future use, so the procedure below is
+one-time only.
+
+To set up a new codespace:
+
+1. In GitHub, click your user avatar and select **Your Codespaces**.
```
Comment (tomekprus) [inline README.md:157]: Not sure if it's me but I don't see 'Your codespaces' option to select.

Comment (pensjonatus) [reply README.md:157]: Widziałem, że niektórzy nie dostają tej opcji.

Dodam uwagę, że trzeba wejść na https://github.com/codespaces

**Review by monBon-Ton — APPROVED** [review]
Comments made outside of github

**Pull request merged**